### PR TITLE
Trivialize and update Florida_Today.xml

### PR DESCRIPTION
**Pull request by cschanaj**
#17965

### CODE REVIEW

File: src/chrome/content/rules/Florida_Today.xml [hunk 1]
```diff
@@ -1,83 +1,46 @@
...
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://ads.flatdy.com/ => http://ads.flatdy.com/: (6, 'Could not resolve host: ads.flatdy.com')
-
-Disabled by https-everywhere-checker because:
-Fetch error: http://ads.flatdy.com/ => http://ads.flatdy.com/: (6, 'Could not resolve host: ads.flatdy.com')
-Fetch error: http://floridatoday.com/ => https://www.floridatoday.com/: (51, "SSL: no alternative certificate subject name matches target host name 'www.floridatoday.com'")
 	For other Gannett Company coverage, see Gannett-Company.xml.
 
+	Non-functional hosts
```
Comment (pipboy96) [inline src/chrome/content/rules/Florida_Today.xml]:
```suggestion
```

Comment (pipboy96) [reply src/chrome/content/rules/Florida_Today.xml]: or add `:` at the end.

Comment (pipboy96) [inline src/chrome/content/rules/Florida_Today.xml]: oops, simply add `:` at the end.

**Comment by pipboy96** [comment]
```
ERROR src/chrome/content/rules/Florida_Today.xml: Fetch error: http://floridatoday.com/ => https://floridatoday.com/: (28, 'Operation timed out after 20001 milliseconds with 0 out of 0 bytes received')
ERROR src/chrome/content/rules/Florida_Today.xml: Fetch error: http://account.floridatoday.com/ => https://account.floridatoday.com/: (28, 'Operation timed out after 20001 milliseconds with 0 out of 0 bytes received')
```
wtf

**Comment by pipboy96** [comment]
Both work for me, redirecting to `eu. floridatoday.com` (I have EU IP).

**Comment by cschanaj** [comment]
> ERROR src/chrome/content/rules/Florida_Today.xml: Fetch error: http://static.floridatoday.com/ => https://static.floridatoday.com/: (28, 'Operation timed out after 20001 milliseconds with 0 out of 0 bytes received')

All `target` work for me indeed. However, another error occurs this time. It seems to me that Travis timeout (randomly) more often in recent builds.

**Comment by pipboy96** [comment]
To be honest, I really don't like whitelisting entire rulesets because of 2 untestable targets, but currently whitelisting individual targets is not possible. Thx for adding `:`.

**Comment by cschanaj** [comment]
Travis passes without whitelisting this time. thanks.

**Comment by cschanaj** [comment]
@pipboy96 since you have done much on the review, would you merge/ approve this PR?

**Pull request merged**